components/eam/src/chemistry/modal_aero/module_mosaic_astem.F90

```diff
@@ -245,11 +245,16 @@ subroutine ASTEM(   mcall_print_aer,
                                 mosaic_vars_aa%fix_astem_negative, aer, gas )
 
   ! condense secondary organic gases (8 sorgam species)
-
+#if ( defined VBS_SOA )
+  ! call ASTEM_VBS_secondary_organics(dtchem,jaerosolstate,sfc_a,Heff,phi_volatile_l,  &
+  !     integrate,aer,kg,gas,sat_soa,total_species)
```

Comment:
@mingxuanwupnnl, is this line of code kept here for the coupling of SOA and MOSAIC in the future?

Reply:
Yes. Currently, we should probably have a new subroutine on doing gas-aerosol partition between SOA aerosol and SOAG gas in MOSAIC in the future.